applications/GeoMechanicsApplication/custom_strategies/strategies/geo_mechanics_newton_raphson_erosion_process_strategy.hpp

```diff
@@ -239,30 +219,26 @@ class GeoMechanicsNewtonRaphsonErosionProcessStrategy
     /// </summary>
     /// <param name="PipeElements"></param>
     /// <returns></returns>
-    int InitialiseNumActivePipeElements(std::vector<Element*> PipeElements)
+    template <typename FilteredElementType>
+    SizeType InitialiseNumActivePipeElements(const FilteredElementType& PipeElements)
```

Comment:
If I read the function, nothing really seems to be initialized, so I'd propose to rename
```suggestion
    SizeType GetNumberOfActivePipeElements(const FilteredElementType& PipeElements)
```

Reply:
Done

Reply:
Done

Reply:
done